contracts/rollup.sol

```diff
@@ -41,19 +41,15 @@ contract Rollup {
     MTUtils public merkleUtils;
 
     bytes32 public constant ZERO_BYTES32 = 0x0000000000000000000000000000000000000000000000000000000000000000;
-    // bytes32 public constant ZERO_MERKLE_ROOT = 0x290decd9548b62a8d60345a988386fc84ba6bc95484008f6362f93160ef3e563;
     address payable constant BURN_ADDRESS = 0x0000000000000000000000000000000000000000;
+    // bytes32 public constant ZERO_MERKLE_ROOT = 0x290decd9548b62a8d60345a988386fc84ba6bc95484008f6362f93160ef3e563;
```

Comment:
I think you can safely remove this, since its not being used anywhere

Reply:
Alright. I will remove this.